### DISCUSSION TITLE
How get total count of channel

### DISCUSSION THREAD

**Discussion by spvince** [GetStream/stream-chat-android#5128] [General]
Hi,

SDK: 6.0.10

I use the code below to get my channels. I would like to know how to get the total count of channels ?

```
ChannelViewModelFactory(
        filters = and(
            eq("type", "messaging"),
            `in`("members", listOf("id of my current user))),
        ),
        querySort = QuerySortByField<Channel>()
            .desc(fieldName = "last_message_at"),
    )
```

**Comment by aleksandar-apostolov** [comment]
Hi @spvince 
Getting the total number of channels is not possible. This number is not computed in the backend and greatly depends on the filters that are provided to the query. 

The SDK has pagination implemented and you can use the information to determine the end of the channel list if that can help.

Let us know if we can further assist you.
